Presentation.Web/app/helpers/connection-change-log-helper.ts

```diff
@@ -0,0 +1,20 @@
+﻿module Kitos.Helpers {
+    export class ConnectionChangeLogHelper {
+        static createDictionaryFromChangeLogList(changeLogs: Array<Models.Api.Organization.ConnectionChangeLogDTO>) {
```

Comment:
please specify the type of the return value... it sure looks like a dictionary of id-> select2object ;)